Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py

```diff
@@ -7,6 +7,7 @@
     ),
     outputCommands = cms.untracked.vstring(
         'keep *_ALCARECOMuAlCalIsolatedMu_*_*', 
+        'keep *_generalTracks_*_*',
```

Comment:
This was a request from Yuriy based on some changes he plans to make to the way the track propagation is handled in the Muon Alignment procedure. I haven't seen his code (and he isn't in the office) so I would hesitate commenting further before speaking to him @pakhotin